src/transformers/models/whisper/tokenization_whisper_fast.py

```diff
@@ -582,14 +582,24 @@ def get_prompt_ids(self, text: str, return_tensors="np"):
         batch_encoding.convert_to_tensors(tensor_type=return_tensors)
         return batch_encoding["input_ids"]
 
-    @staticmethod
-    # Copied from transformers.models.whisper.tokenization_whisper.WhisperTokenizer._strip_prompt
-    def _strip_prompt(token_ids: List[int], prompt_token_id: int, decoder_start_token_id: int):
+    def _strip_prompt(self, token_ids: List[int], prompt_token_id: int, decoder_start_token_id: int):
         has_prompt = isinstance(token_ids, list) and token_ids and token_ids[0] == prompt_token_id
+        if not isinstance(token_ids, list):
+            token_ids = self._convert_to_list(token_ids)
         if has_prompt:
             if decoder_start_token_id in token_ids:
                 return token_ids[token_ids.index(decoder_start_token_id) :]
             else:
                 return []
 
         return token_ids
+
+    @staticmethod
+    def _convert_to_list(token_ids):
```

Comment:
Let's copy this method from `tokenization_whisper.py`, as suggested [here](https://github.com/huggingface/transformers/pull/27818/files#r1634779349):

```suggestion
    @staticmethod
    # Copied from transformers.models.whisper.tokenization_whisper.WhisperTokenizer. _convert_to_list
    def _convert_to_list(token_ids):
```